fe/src/main/java/org/apache/doris/http/rest/GetLoadInfoAction.java

```diff
@@ -63,7 +64,15 @@ public void executeWithoutPassword(AuthorizationInfo authInfo, BaseRequest reque
         if (redirectToMaster(request, response)) {
             return;
         }
-        catalog.getLoadInstance().getJobInfo(info);
+        try {
+            try {
+                catalog.getLoadInstance().getJobInfo(info);
+            } catch (DdlException e) {
+                catalog.getLoadManager().getLoadJobInfo(info);
```

Comment:
I think you should first get info from your new Load Manager, then from the old Load Instance

Reply:
What‘s the difference between those two?

Reply:
LoadManager is your new implementation, so only history load job may exist in LoadInstance.
If you check LoadInstance first, then you have to check both LoadInstance and LoadManager for every load job info request in the future, which is inefficient.

Reply:
The hadoop load job still exists in the LoadInstance. In the future, it must check both LoadInstance and LoadManager for every load job info request. So the sequence is not important.